actions/castVote.ts

```diff
@@ -73,9 +75,29 @@ export async function castVote(
     tokenOwnerRecord
   )
 
+  // If the proposal is multi-choice
+  const approveChoices: VoteChoice[] = [];
```

Comment:
let's rewrite this as immutable code using `filter` and `map`.

Reply:
done